docs/pages/guides/modes.mdx

```diff
@@ -0,0 +1,82 @@
+import { Markdown } from '../../components/Page';
+
+# Modes
+
+With the support of SQLite in Keystone 6, you can now choose to use Keystone in a “standalone” (headless) fashion, or “embed” your backend within a frontend app.
```

Comment:
```suggestion
With the support of SQLite in Keystone Next, you can now choose to use Keystone in a “standalone” (headless) fashion, or “embed” your backend within a frontend app.
```

**SO CLOSE** but we're not there yet 😛

Reply:
Yeah, no worries! I thought that wouldn't make the cut. More of a bit of a mental note, and to try it on for size.